config/assisted-service/scality-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: assisted-installer-s3
+  namespace: "assisted-installer"
+  labels:
+    app: scality
+type: Opaque
+stringData:
+  aws_access_key_id: "accessKey1"
+  aws_region: "us-east-1"
+  aws_secret_access_key: "verySecretKey1"
```

Comment:
Same here, can we generate this?

Reply:
In this case it doesn't matter as much, because a follow-up PR will remove S3 involvement